content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -34,7 +34,78 @@ You must run the RDI installer as a privileged user because it installs
 [containerd](https://containerd.io/) and registers services. However, you don't
 need any special privileges to run RDI processes for normal operation.
 
-### Hardware sizing
+The [K3s](https://k3s.io/) Kubernetes distribution used internally by RDI has a few
+requirements for cloud VMs that you must implement before running the
+RDI installer, or else installation will fail. The following sections
+give full pre-installation instructions for [RHEL](#k3s-rhel) and
+[Ubuntu](#k3s-ubuntu).
+
+### RHEL {#k3s-rhel}
+
+K3s recommends that you turn off 
+[`firewalld`](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/using-and-configuring-firewalld_configuring-and-managing-networking)
+before installation using the command:
+
+```bash
+systemctl disable firewalld --now
+```
+
+However, if you do need to use `firewalld`, you must add the following rules:
+
+```bash
+firewall-cmd --permanent --add-port=6443/tcp #apiserver
+firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16 #pods
+firewall-cmd --permanent --zone=trusted --add-source=10.43.0.0/16 #services
+firewall-cmd --reload
+```
+
+You may also need to open other ports if your setup requires them. See the K3s
+[Inbound rules](https://docs.k3s.io/installation/requirements?_highlight=red&_highlight=hat&os=rhel#inbound-rules-for-k3s-nodes)
+docs for more information. If you change the default CIDR for pods or services,
+you must update the firewall rules accordingly.
+
+If you have `nm-cloud-setup.service` enabled, you must disable it and reboot the
+node with the following commands:
+
+```bash
+systemctl disable nm-cloud-setup.service nm-cloud-setup.timer
+reboot
+```
+
+See
+[Rancher support and maintenance terms](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-8-6/)
+for more information about the OS versions that have been tested with Rancher
+managed K3s clusters.
```

Comment:
Seems like is should be "...Rancher-managed..." in this context? Feel free to ignore.

Reply:
Well spotted! This is actually adapted from the K3s docs - I just copied and pasted this bit without reading it carefully enough.